aptos-move/framework/aptos-framework/sources/lite_account.move

```diff
@@ -0,0 +1,204 @@
+module aptos_framework::lite_account {
```

Comment:
maybe we should have accounts folder, and have this in there - and name it `move_authenticator.move`, 'dispatchable_authenticator.move' or `account_abstraction.move` (maybe move account.move to that folder too)

nothing in this PR needs to use term "lite", and actual "lite" structs should probably go to separate file anyways - they will just be stored at the same address.

Reply:
kk. doing that now.